azext_iot/_help.py

```diff
@@ -1173,7 +1173,7 @@
     "iot central"
 ] = """
     type: group
-    short-summary: Manage Azure IoT Central assets.
+    short-summary: Manage Azure IoT Central assets. To use this command group, the user must be logged through the `az login` command, have the correct tenant set (the users home tenant) and have access to the application through http://apps.azureiotcentral.com"
```

Comment:
I would recommend using a pipe | to organize the short-summary over multiple lines. Example:

![image](https://user-images.githubusercontent.com/28658112/79916640-ef0f7480-83dd-11ea-9e1e-3e30f7338a1b.png)

Reply:
Actually, I cant get this help message to show.  Could the Azure CLI core help message be overwriting it?

Reply:
Hmm - usually its the other way around, CLI extensions are able to override/replace CLI core content.  I'll see if I can repro.